mainapp/models.py

```diff
@@ -140,7 +142,7 @@ class NGO(models.Model):
     organisation_type = models.CharField(max_length=250, verbose_name="Type of Organization")
     organisation_address = models.TextField(default='', verbose_name="Address of Organization")
     name = models.CharField(max_length=100, verbose_name="Contact Person")
-    phone = models.CharField(max_length=10)
+    phone = models.CharField(max_length=12)
```

Comment:
This change is also being made in https://github.com/IEEEKeralaSection/rescuekerala/pull/460, So it could be removed from here.

Also, please push your migrations file for the changes you've made in the model.

Reply:
@sks444 I think we should accept with 14 too

Reply:
Fixed in https://github.com/IEEEKeralaSection/rescuekerala/pull/480